src/types.rs

```diff
@@ -56,6 +65,12 @@ pub struct QuarantineBulkTestStatus {
     pub quarantine_results: Vec<QuarantineResult>,
 }
 
+#[derive(Debug, Serialize, Clone, Deserialize)]
+pub struct TrunkRepo {
```

Comment:
If we aren't using response object, maybe we can have empty schema for now.

Reply:
Yeah, we aren't using it, but I think it's fine to leave this as a sanity check that the repoId exists in the response